### ISSUE TITLE
MW 1.39 release issue tracker

### ISSUE THREAD

**Issue by yaronkoren** [CanastaWiki/Canasta#162]
It's my hope that the MediaWiki 1.39 version of Canasta will not just be another version - with updated MediaWiki code - but the first truly stable version, where major bugs and problems no longer exist. (One can hope, anyway.) To that end, I've compiled a list of all the bugs/issues that I feel need to be fixed/addressed before the official 1.39 version of Canasta can be released. I've also listed a bunch of issues that would be nice to fix, but I don't think should block the release if they're not done.

It should be noted that some of these issues (of both types) already have patches created for them that just need to be reviewed and (ideally) merged in.

Tasks with a purple icon have been merged into master or some branch, and should be considered completed.

Tasks with a dagger (**†**) at the end have a patch created for them that has not yet been approved.

## Main Canasta issues
- #145 
- #148
- #197
- #198

### Nice to have but not necessary
- #85
- #139
- #159 **†**
- https://github.com/CanastaWiki/Canasta-DockerCompose/pull/26

## CLI issues
- https://github.com/CanastaWiki/Canasta-CLI/issues/26 **†**
- https://github.com/CanastaWiki/Canasta-CLI/issues/32
- https://github.com/CanastaWiki/Canasta-CLI/issues/42
- https://github.com/CanastaWiki/Canasta-CLI/issues/37
- https://github.com/CanastaWiki/Canasta-CLI/issues/40

### Nice to have but not necessary
- https://github.com/CanastaWiki/Canasta-CLI/issues/43
- https://github.com/CanastaWiki/Canasta-CLI/issues/41 **†**
- https://github.com/CanastaWiki/Canasta-CLI/pull/36  **†**

## Extension and skin issues
- #137
- #107
- #181
- #143
- #155
- #163
- #166
- #167 
- #168

### Nice to have but not necessary
- #176 
- Add SimpleSAMLphp extension (may be contingent on change to Traefik)
- #152 
- #172
- #177

**Comment by tosfos** [comment]
Linking to  #141

**Comment by jeffw16** [comment]
If I understand the conversation @yaronkoren and I had offline, we should be able to ship out Canasta 1.3.0 (which runs on MediaWiki 1.39) as long as the main issues are resolved, except if an extension is taking too long to gain MediaWiki 1.39 support and we're unable to help them make the necessary updates.

**Comment by FelipoAntonoff** [comment]
Hi, any news about the new version with MediaWiki 1.39 support and Traefik support?

**Comment by jeffw16** [comment]
Canasta 1.3.0 released. Closing
